src/js/components/award/AgencyInfo.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * AgencyInfo.jsx
+ * Created by Emily Gullo 01/20/2017
+ **/
+
+import React from 'react';
+import InfoSnippet from './InfoSnippet';
+
+const propTypes = {
+    selectedAward: React.PropTypes.object,
+    params: React.PropTypes.object
+};
+
+export default class AgencyInfo extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            agencyType: "awarding"
+        };
+
+        this.toggleAgency = this.toggleAgency.bind(this);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.selectedAward.id !== this.props.params.awardId) {
```

Comment:
This component doesn't have props named `params`. The only prop is `selectedAward`; you'll either need to pass the URL params down or compare `selectedAward.id` values.